Project-1/README.md

```diff
@@ -11,56 +11,276 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+	
+	
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+  ```
+  (a) ∃x∀y and ∀x∃y
+  	** These read "There exists some X such that for all y..." and "For all x there exists some y such that..." There is no way to explain the difference beyond this because these are incomplete sentences. They are analogous to "sentence fragments" from natural language where the Subject was missing and the only verb is "is"; "Is large and blue."  **
+	
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  	** These read "There exists some X such that for all Y and some z..." and "For all X there exists some Y such that all Z. .. " There is no way to explain the difference beyond this because these are incomplete sentences **
+	
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  	** These read "For all X there exists some Y such that all Z and some W. . ." and "There exists some X for all Y such that some some Z and all W . . . " There is no way to explain the difference beyond this because these are incomplete sentences.  **
+  
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+```
+ {(a,b)(b,a)(a,a)} 
+ The second statement is true in this model. There is an x that relates every y (a, related to a and b) There is an x that relates to a z (where a is x and b is z), all ys (a and b) are related to some z (a is related to a and b, b is related to a)
+ The first statement is not true. It is not true that all xs (a and b) are related to all zs (b is not related to a)
+
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
-  ```(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  ```
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  ![proof(1)](https://user-images.githubusercontent.com/123899465/216850416-7fb691da-f18f-4164-877c-bc1064085130.png)
+
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  ![proof(2)](https://user-images.githubusercontent.com/123899465/216850442-db0062d8-5ffb-4e46-bec6-1135c47a53a8.png)
+
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-  (d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  ![proof(3)](https://user-images.githubusercontent.com/123899465/216850473-a00e0b27-ee52-43a3-a2b8-e7060cd0abac.png)
+
+   (d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+    ![proof(4)](https://user-images.githubusercontent.com/123899465/216850506-16aea313-2ca2-4297-8615-668f8d5a3701.png)
+
 ```
 	
-9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws.
+a) 
+assume -(Px & Qx).
```

Comment:
You didn't instantiate the variable x to anything. 

You didn't provide justification of lines in the proofs...